packages/2d/src/components/Txt.ts

```diff
@@ -105,6 +105,28 @@ export class Txt extends Shape {
     context.restore();
   }
 
+  protected override getCacheBBox(): BBox {
+    const cache = super.getCacheBBox();
+
+    if (!this.element.hasChildNodes) return cache;
```

Comment:
One final thing, these lines shouldn't be necessary now because we no longer use children for selection.
Additionally, `hasChildNodes` is a function, not a property, so right now this if statement is always false anyway.